### PR TITLE
Update Microsoft.DotNet.TestHost package version.

### PR DESCRIPTION
**Pull request by pallavit**
TestHost package has been updated to include a new APISet. This change updates the BuildTools to reference the new package.

### CODE REVIEW

**Comment by pallavit** [comment]
@weshaggard @AlexGhiondea Could you please review this change.

File: src/nuget/Microsoft.DotNet.BuildTools.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata minClientVersion="2.8.1">
     <id>Microsoft.DotNet.BuildTools</id>
-    <version>1.0.25-prerelease</version>
+    <version>1.0.26-prerelease</version>
```
Comment (AlexGhiondea) [inline src/nuget/Microsoft.DotNet.BuildTools.nuspec]: I am not sure if this is needed.. @jhendrixMSFT do you know?

Comment (jhendrixMSFT) [inline src/nuget/Microsoft.DotNet.BuildTools.nuspec]: It is not.  The daily build will append a build number so there is no need to bump patch number at this time.

Comment (pallavit) [inline src/nuget/Microsoft.DotNet.BuildTools.nuspec]: Thanks. Undid the changes to this file and rebased.

**Comment by AlexGhiondea** [comment]
LGTM

**Comment by pallavit** [comment]
Thanks for the review.

**Pull request merged**